project/make/.dockernsenter

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I'd much rather we just rename `project/make/.dockerversion` to `project/make/.autogen` and put all the autogenerated code in one place (and thus this file into `autogen/...` and imported there from `docker/...` instead).